resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -327,22 +327,37 @@ function task_preinst_deb {
 <% end -%>
 }
 
-# Debian based post-installation tasks.
+# Debian based unconditional post-installation tasks.
 function task_postinst_deb {
     task postinst_permissions
 <% EZBake::Config[:debian][:additional_postinst].each do |cmd| -%>
     <%= cmd %>
 <% end -%>
 }
 
-# RPM based post-installation tasks.
+# Debian based install post-installation tasks.
+function task_postinst_deb_install {
+<% EZBake::Config[:debian][:additional_postinst_install].each do |cmd| -%>
+    <%= cmd %>
+<% end -%>
```

Comment:
You need a null command in here now that you've removed the postinst_permissions